draft-wussler-openpgp-pqc.md

```diff
@@ -564,7 +564,15 @@ As explained in {{non-composite-multi-alg}}, the OpenPGP protocol inherently
 supports parallel encryption to different keys of the same recipient.
 Implementations MUST NOT encrypt a message to a purely traditional public-key
 encryption key of a recipient if it is encrypted to a PQ/T key of the same
-recipient.
+recipient. This can be achieved with checks at different levels, for instance:
+
+- By ensuring only one key per recipient is used.
+
+- By checking if there is a "Superseded by" signature that links to another
```

Comment:
This seems to refer to this proposal: https://gitlab.com/openpgp-wg/rfc4880bis/-/issues/64

But this is not yet part of the crypto-refresh. So I would prefer not to include it here until it becomes adapted into the cr-re